lib/postoffice/messaging.ex

```diff
@@ -120,7 +134,24 @@ defmodule Postoffice.Messaging do
   def create_publisher_success(attrs \\ %{}) do
```

Comment:
We should rename this to `build_publisher_success` because as we are using the ecto magician to do all transactionally, I removed the `insert`

What do you think about open an issue with this?